plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -209,15 +210,15 @@ private JdbcTableHandle flushAttributesAsQuery(ConnectorSession session, JdbcTab
                 .map(JdbcColumnHandle.class::cast)
                 .collect(toImmutableList());
 
-        Set<JdbcColumnHandle> newColumnSet = ImmutableSet.copyOf(newColumns);
-
         if (handle.getColumns().isPresent()) {
+            Set<JdbcColumnHandle> newColumnSet = ImmutableSet.copyOf(newColumns);
             Set<JdbcColumnHandle> tableColumnSet = ImmutableSet.copyOf(handle.getColumns().get());
             if (newColumnSet.equals(tableColumnSet)) {
                 return Optional.empty();
             }
 
-            verify(tableColumnSet.containsAll(newColumnSet), "applyProjection called with columns not available in existing query");
+            verify(tableColumnSet.containsAll(newColumnSet),
+                    "applyProjection called with columns %s and some are not available in existing query: %s", newColumnSet, tableColumnSet);
```

Comment:
fmt -- all args on one line, or each on sepearate line (including the first one)